adoc/TRD-SLES-SAP-HA-automation-quickstart-cloud.adoc

```diff
@@ -16,19 +16,22 @@ endif::[]
 // enable it for editor previews
 //:cloud: Azure
 //:cloud: AWS
-//:cloud: GCP
+:cloud: GCP
 
 :sles: SUSE Linux Enterprise Server
 :sles4sap: {sles} for SAP applications
 :hana_version: SAP HANA 2.0 SPS05
 :hana_archive_version: 51054623
+:gcp_par_count: 13
+:aws_par_count: 10
+:azure_par_count: 13
```

Comment:
i did not like the source sentence where this is needed. i think nobody cares if the 1o,11,12,13 steps - is not a information which is necessary or needed to setup the solution

Reply:
My goal is to show the guide reader how easy it is. You need to update 13 parameters only to create SAP HANA HA Cluster. 
If you or Stephen does not like it, I can easily set it for Google Cloud only. ;)